### PR TITLE
Limit block range

### PR DESCRIPTION
**Pull request by pathornteng**
- [ ] I added unit tests for any code that added
- [ ] I updated the CHANGELOG.md 
- [x] All IP is original and not copied from another source
- [x] I assign all copyright to Loom Network for the code in the pull request

### CODE REVIEW

File: rpc/eth/config.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package eth
+
+type Web3Config struct {
+	// MaxBlockLimit defines the maximum number of block range in one request
+	MaxBlockLimit int64
```
Comment (enlight) [inline rpc/eth/config.go]: Lets use a more specific name, `GetLogsMaxBlockRange`

Comment (pathornteng) [reply rpc/eth/config.go]: Done

File: evm/config.go [hunk 2]
```diff
@@ -24,6 +24,17 @@ func DefaultEvmStoreConfig() *EvmStoreConfig {
 	}
 }
 
+type EvmConfig struct {
```
[the remark this answers is not in the document]
Comment (pathornteng) [reply evm/config.go]: Done

File: config/config.go [hunk 3]
```diff
@@ -143,6 +143,8 @@ type Config struct {
 	EVMDebugEnabled bool
 	// Set to true to disable minimum required build number check on node startup
 	SkipMinBuildCheck bool
+
+	Evm *evm.EvmConfig
```
[the remark this answers is not in the document]
Comment (pathornteng) [reply config/config.go]: Done

**Pull request merged**